build.gradle

```diff
@@ -704,6 +704,8 @@ uiTestTargets.each { version ->
 
     if (project.hasProperty('headless')) {
       environment 'IDEPROBE_DISPLAY', 'xvfb'
+      environment 'IDEPROBE_TEST_SUITE', 'uiTest'
```

Comment:
Pls add a TODO (see #924 for the format) to a corresponding issue in ide-probe (pls create), as this should theoretically NOT be needed IIUC

Reply:
yes now I am testing all cases with this environments and I will make an issue in ide-probe